### DISCUSSION TITLE
dynamic Loglevel  change without restarting app

### DISCUSSION THREAD

**Discussion by srinivasmummareddy** [open-telemetry/opentelemetry-dotnet#5360] [Q&A]
Is it possible to  change loglevel  dynamically without restarting app similar to nlog and serilog and writing logs to files?

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/issues/5513 An issue that might occur with reload.
(To your original question, yes, it is supported by ILogger itself, not necessarily an OTel specific feature)
